frontend/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -15,11 +13,19 @@ const PopupWithForm = (props) => {
           }}
           className="popup__block"
         >
+ <form
```

Comment:
You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```jsx

const PopupWithForm = (props) => {
  return (
    <div
      onClick={props.onClose}
      className={props.openPopup ? "popup popup_open" : "popup"}
    >
      <div
        onClick={(evt) => {
          //cancel onClick func on that div
          evt.stopPropagation();
        }}
        className="popup__block"
      >
        <form
          className="popup__form"
          onSubmit={props.onSubmit}
        >
          <button
            type="button"
            onClick={props.onClose}
            className="popup__close-btn"
          ></button>
          {/* here goes form */}
          {props.children}
          {/* here go the span part  */}
        </form>
      </div>
    </div>
  );
};


```